snomed/com.b2international.snowowl.snomed.api.impl/src/com/b2international/snowowl/snomed/api/impl/SnomedReleases.java

```diff
@@ -33,19 +26,43 @@
  */
 public class SnomedReleases {
 
-	public static CodeSystem newSnomedInternationalRelease() {
+	private static final CodeSystem INT_CODESYSTEM = CodeSystem.builder()
+			.name(SNOMED_NAME)
+			.shortName(SNOMED_SHORT_NAME)
+			.oid(SNOMED_INT_OID)
+			.primaryLanguage(SNOMED_INT_LANGUAGE)
+			.organizationLink(SNOMED_INT_LINK)
+			.citation(SNOMED_INT_CITATION)
+			.branchPath(IBranchPath.MAIN_BRANCH)
+			.iconPath(SNOMED_INT_ICON_PATH)
+			.repositoryUuid(REPOSITORY_UUID)
+			.terminologyId(TERMINOLOGY_ID)
+			.build();
+
+	/**
+	 * @return the code system representing SNOMED CT's International release
+	 */
+	public static CodeSystem internationalRelease() {
+		return INT_CODESYSTEM;
+	}
+
+	/**
+	 * @return the code system representing the B2i extension of SNOMED CT
+	 */
+	public static CodeSystem b2iExtension(String extensionPath) {
 		return CodeSystem.builder()
-				.name(SNOMED_INT_NAME)
-				.shortName(SNOMED_INT_SHORT_NAME)
-				.oid(SNOMED_INT_OID)
+				.name("Systematized Nomenclature of Medicine Clinical Terms, B2i extension")
```

Comment:
Like CDO, SNOMED is no longer officially an acronym. I guess this should be **SNOMED CT, B2i extension**